SPECS/asciidoc/asciidoc.spec

```diff
@@ -60,12 +60,15 @@ python3 tests/testasciidoc.py run
 %{_mandir}/*
 %{python3_sitelib}/asciidoc/
 %{_pkgdocdir}/doc
-%{python3_sitelib}/asciidoc-*.egg-info
+%{python3_sitelib}/asciidoc-*
+
 %dir %{python3_sitelib}/asciidoc/resources/filters/latex
 %dir %{python3_sitelib}/asciidoc/resources/filters/music
 
 
 %changelog
```

Comment:
Optional: I would leave the convention of `space-separated-changelog` as your call :)

Reply:
sure, will add the spaces to match other specs